src/Mvc/Mvc.Testing/src/WebApplicationFactory.cs

```diff
@@ -199,34 +198,6 @@ private string GetContentRootFromFile(string file)
             return (contentRoot == "~") ? AppContext.BaseDirectory : contentRoot;
         }
 
-        private string GetContentRootFromAssembly()
```

Comment:
I forget @javiercn do we need to keep the attribute around for back compat or should I nuke the attribute and relevant code now as part of this cleanup PR?

Reply:
I'm hesitant about it. It's technically a breaking change, but I can't tell how many people is using it. They would have to switch to provide their own config file at that point.

Reply:
Ok i will just leave the code and undo these changes